test/built-ins/NaN/S15.1.1.1_A1.js

```diff
@@ -2,22 +2,8 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-info: The initial value of NaN is NaN
 es5id: 15.1.1.1_A1
-description: Use typeof, isNaN, isFinite
+description: The initial value of NaN is NaN
 ---*/
 
-// CHECK#1
-if (typeof(NaN) !== "number") {
-	$ERROR('#1: typeof(NaN) === "number". Actual: ' + (typeof(NaN))); 
-}
-
-// CHECK#2
-if (isNaN(NaN) !== true) {
-	$ERROR('#2: NaN === Not-a-Number. Actual: ' + (NaN)); 
-}
-
-// CHECK#3
-if (isFinite(NaN) !== false) {
-	$ERROR('#3: NaN === Not-a-Finite. Actual: ' + (NaN)); 
-}
+assert(NaN !== NaN);
```

Comment:
As noticed in #713, we already have tests for Number.NaN which is already asserted as a NaN value. So it does not really matter checking typeof for NaN and Number.NaN.

In any case, I replaced the namespaced names with their equivalent globals.